geoipbackend: no previous declaration for validateMappingLookupFormats

<!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
<!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on IRC (https://www.powerdns.com/opensource.html) -->

<!-- Tell us what is issue is about -->
 - Program: Authoritative
 - Issue type: compile bug report

### Short description
<!-- Explain in a few sentences what the issue/request is -->

### Environment
<!-- Tell us about the environment -->
 - Operating system: Debian unstable
 - Software version: 4.4.0-beta1
 - Software source: tarball

### Steps to reproduce
<!-- Tell us step-by-step how the issue can be triggered. Please include your configuration files and any (Lua) scripts that are loaded. -->
1. try a debian build
2. observe the warning emitted by gcc

### Actual behaviour
<!-- What did happen? Please (if possible) provide logs, output from `dig` and/or tcpdump/wireshark data -->

```
geoipbackend.cc:99:6: warning: no previous declaration for ‘bool validateMappingLookupFormats(const std::vector<std::__cxx11::basic_string<char> >&)’ [-Wmissing-declarations]
   99 | bool validateMappingLookupFormats(const vector<string>& formats) {
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### Other information
<!-- if you already did more digging into the issue, please provide all the information you gathered -->

```
g++ (Debian 10.2.0-19) 10.2.0
```

```
/bin/bash ../../libtool  --tag=CXX   --mode=compile g++ -DHAVE_CONFIG_H -I. -I../..  -I../.. -I../.. -pthread   -I/usr/include/ -I/usr/include/x86_64-linux-gnu -Wdate-time -D_FORTIFY_SOURCE=2  -fPIE -DPIE -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 --param ssp-buffer-size=4 -fstack-protector -std=c++11 -g -O2 -Wall -Wextra -Wshadow -Wno-unused-parameter -Wmissing-declarations -Wredundant-decls -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -DPACKAGEVERSION='"4.4.0~beta1-1.Debian"' -c -o geoipbackend.lo geoipbackend.cc
```

--- a/modules/geoipbackend/geoipbackend.cc
+++ b/modules/geoipbackend/geoipbackend.cc
@@ -96,7 +96,7 @@ static string queryGeoIP(const Netmask& addr, GeoIPInterface::GeoIPQueryAttribut
 // validateMappingLookupFormats validates any custom format provided by the
 // user does not use the custom mapping placeholder again, else it would do an
 // infinite recursion.
-bool validateMappingLookupFormats(const vector<string>& formats) {
+static bool validateMappingLookupFormats(const vector<string>& formats) {
   string::size_type cur,last;
   for (const auto& lookupFormat : formats) {
     last=0;

